### PR TITLE
Add example for deleting hooks

### PR DESCRIPTION
**Pull request by anatrajkovska**
This PR adds an example that handles deleting hooks that have set a Delete Hooks URI.

Documentation PR https://github.com/zeit/docs/pull/890.

### CODE REVIEW

File: delete-hooks-redirect-uri/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+npm install
+```
+
+Then run this UiHook locally with:
+
+```
+now dev -p 5005
+```
+
+Then [create a ZEIT integration](https://zeit.co/docs/integrations) and set the Redirect URI to: `http:///localhost:5005`
```
Comment (arunoda) [inline delete-hooks-redirect-uri/README.md]: We need to expand this section.

```suggestion
## Create the Integration

Create an integration by logging in to the [Integration Console](https://zeit.co/dashboard/integrations/console).

Then set the "Redirect URL" to `http://localhost:5005`.

Leave the "UIHook URL" field blank.
```

File: delete-hooks-redirect-uri/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	`;
+});
+
+module.exports = function(req, res) {
+  if (req.method === 'DELETE') {
+    console.log('Configuration has been deleted. Do some cleanup tasks here.');
+    return micro.send(res, 200, {});
+  }
+
+  return renderUIHook(req, res);
```
Comment (arunoda) [inline delete-hooks-redirect-uri/index.js]: Do not add any UIHook stuff here. 
Add a console log saying "Configuration has added".

Do this: https://github.com/zeit/integrations/blob/master/webhooks-explorer/index.js#L67-L69

So, right after we add the configuration it'll redirect to the ZEIT dashboard.

File: delete-hooks-redirect-uri/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# DELETE Hooks - Redirect URI example
+
+Integrations that require OAuth credentials need to set a Redirect URI. If a user removes a configuration for such an integration sometimes is useful to do some cleanup tasks.
```
Comment (arunoda) [inline delete-hooks-redirect-uri/README.md]:
```suggestion
This is a ZEIT Integration using the Redirect URI. This example is very based an do nothing other than printing some messages to the console.

But it shows how you can handle the DELETE Hook request when the user remove the Integration configuration.
Using that, integration can implement clean up tasks like deleting created databases and so on.
```

**Comment by anatrajkovska** [comment]
@arunoda Can you review the PR again?

File: delete-hooks/index.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  const { query } = parseUrl(req.url, true);
+  if (req.method === 'DELETE') {
+    const payload = await micro.json(req)
+    console.log('Configuration has been deleted. You can do some cleanup tasks here.');
+		console.log(`Payload: ${JSON.stringify(payload)}`);
+    return micro.send(res, 200, {});
+  }
+
+  console.log('Configuration has been added.');
+	// Here you can get an ZEIT token an use it to access our APIs.
```
[the remark this answers is not in the document]
Comment (anatrajkovska) [reply delete-hooks/index.js]: @arunoda Fixed.

**Comment by anatrajkovska** [comment]
@arunoda Can you please review?

**Comment by arunoda** [comment]
@anatrajkovska updated the PR. 
Now merging.

**Pull request merged**